netlify/functions/manage-findings.ts

```diff
@@ -90,44 +90,85 @@ async function getFinding(
   };
 }
 
+async function getTeamsFindings(
+  teamNames: string[],
+  octokit: Octokit,
+  contest: Contest,
+  wardenFindingsForContest: WardenFindingsForContest
+): Promise<void[]> {
+  const promises: Promise<void>[] = [];
+  for (const teamName of teamNames) {
+    promises.push(
+      getSingleTeamFindings(teamName, octokit, contest).then((teamFindings) => {
+        wardenFindingsForContest.teams[teamFindings.teamName] =
+          teamFindings.findings;
+      })
+    );
+  }
+  return await Promise.all(promises);
```

Comment:
This is okay but might be better more just inline:

```
return await Promise.all(
  teamNames.map((teamName) => getSingleTeamFindings(teamName, octokit, contest).then(/* etc. */))
);
```

…but I'm seeing something potentially more notable as I look more closely, will add a comment above next.